deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -139,37 +139,56 @@
    (vector? block)
    (= "Timestamp" (first block))))
 
+(defn- get-page-refs-from-property-names
+  [properties {:property-pages/keys [enabled? excludelist]}]
+  (if (contains? #{true nil} enabled?)
+    (some->> properties
+             (map (comp name first))
+             (remove string/blank?)
+             (remove (set (map name excludelist)))
+             ;; Remove built-in properties as we don't want pages
+             ;; created for them by default
+             (remove (set (map name (into (gp-property/editable-built-in-properties)
+                                          (gp-property/hidden-built-in-properties)))))
+             distinct)
+    []))
+
 (defn- get-page-ref-names-from-properties
-  [format properties]
+  [format properties user-config]
   (let [page-refs (->>
-                     properties
-                     (remove (fn [[k _]]
-                               (contains? #{:background-color :background_color} (keyword k))))
-                     (map last)
-                     (map (fn [v]
-                            (cond
-                              (and (string? v)
-                                   (not (gp-mldoc/link? format v)))
-                              (let [v (string/trim v)
-                                    result (text/split-page-refs-without-brackets v {:un-brackets? false})]
-                                (if (coll? result)
-                                  (map text/page-ref-un-brackets! result)
-                                  []))
-
-                              (coll? v)
-                              (map (fn [s]
-                                     (when-not (and (string? v)
-                                                    (gp-mldoc/link? format v))
-                                       (text/page-ref-un-brackets! s))) v)
-
-                              :else
-                              nil)))
-                     (apply concat))
-        property-keys-page-refs (some->> properties
-                                         (map (comp name first))
-                                         (remove string/blank?)
-                                         (distinct))]
-    (->> (concat page-refs property-keys-page-refs)
+                   properties
+                   (remove (fn [[k _]]
+                             (contains?
+                              (set (into (gp-property/editable-built-in-properties)
+                                         (gp-property/hidden-built-in-properties)))
+                              (keyword k))))
```

Comment:
Value of `title`, `filters`, and other built-in properties are all parsed as pages, so I filter them out here.

Reply:
Good catch! I moved these linkable properties up to a var as those same properties are referenced for a behavior that is specific to them